test/sql/function/string/test_levenshtein.test

```diff
@@ -0,0 +1,22 @@
+# name: test/sql/function/string/test_levenshtein.test
+# description: Test levenshtein function
+# group: [string]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE strings(s VARCHAR)
+
+statement ok
+INSERT INTO strings VALUES ('hello'), ('hallo'), ('aloha'), ('hoi'), ('world')
+
+# normal length
+query I
+SELECT levenshtein(s, 'hallo') FROM strings ORDER BY s
```

Comment:
I would like to see a number of extra tests, including `NULL` values, and each of the different combinations of (CONSTANT, CONSTANT), (COLUMN, CONSTANT), (CONSTANT, COLUMN), (COLUMN, COLUMN).

In addition, I would like to see tests that trigger each of the possible error conditions (too short strings, empty strings, too long strings).

Reply:
Noted.